### PR TITLE
PG: Support for multiple projection in aggregate

### PR DESCRIPTION
**Pull request by dplewis**
Adds support for multiple keys in aggregate queries.

Improves Projection Tests
Lints PG adapter.

### CODE REVIEW

File: src/Adapters/Storage/Postgres/PostgresStorageAdapter.js [hunk 1]
```diff
@@ -1546,7 +1545,7 @@ export class PostgresStorageAdapter {
       }
     }
 
-    const qs = `SELECT ${columns} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern} ${groupPattern}`;
```
Comment (vitaly-t) [inline src/Adapters/Storage/Postgres/PostgresStorageAdapter.js]: Are you sure you do not need to join the column there? Or is it the default for ES6 template strings when it comes to arrays?

Comment (vitaly-t) [reply src/Adapters/Storage/Postgres/PostgresStorageAdapter.js]: Did I reject this? Sorry, I'm not quite sure what I'm doing when it comes to reviews :)

Comment (dplewis) [reply src/Adapters/Storage/Postgres/PostgresStorageAdapter.js]: I did join the columns here. They weren't joined before.

File: src/Adapters/Storage/Postgres/PostgresStorageAdapter.js [hunk 2]
```diff
@@ -1546,7 +1545,7 @@ export class PostgresStorageAdapter {
...
+    const qs = `SELECT ${columns.join()} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern} ${groupPattern}`;
```
Comment (dplewis) [inline src/Adapters/Storage/Postgres/PostgresStorageAdapter.js]: @vitaly-t Joined here

**Pull request merged**